packages/components/pie-switch/test/component/pie-switch.spec.ts

```diff
@@ -240,21 +242,21 @@ test.describe('Component: `Pie switch`', () => {
 
     test.describe('Props: `LabelProps`', () => {
         labelPlacements.forEach((labelPlacement) => {
-            test(`should render a ${labelPlacement} label`, async ({ mount }) => {
+            test(`should render a ${labelPlacement} label`, async ({ page }) => {
                 // Arrange
-                const component = await mount(PieSwitch, {
-                    props: {
-                        label: 'Label',
-                        labelPlacement,
-                    } as SwitchProps,
-                });
+                const switchPage = new BasePage(page, 'switch');
+                const props: Partial<SwitchProps> = {
+                    label: 'Label',
+                    labelPlacement,
+                };
 
-                const selector = switchLabelSelector(labelPlacement);
-                const pieSwitchLabel = component.locator(selector);
-                const testId = await pieSwitchLabel.getAttribute('data-test-id');
+                await switchPage.load({ ...props });
+
+                const labelSelector = page.getByTestId(pieSwitch.selectors.label[labelPlacement].dataTestId);
+                const testId = await labelSelector.getAttribute('data-test-id');
 
                 // Assert
-                await expect(pieSwitchLabel).toBeVisible();
+                await expect(labelSelector).toBeVisible();
                 expect(testId).toContain(labelPlacement);
```

Comment:
This last expect can be removed. If it wasn't true, the `page.getByTestId` above would timeout.